### ISSUE TITLE
Not able to render 3d object using modelrenderable below minimum sdk version 24

### ISSUE THREAD

**Issue by SudarshanSudhan** [google-ar/sceneform-android-sdk#163]
> SceneView can now be used instead of ArSceneView to render a scene when not using ARCore.
>     The camera can be moved and rotated via SceneView.getScene().getCamera().
>         The initial camera transform is currently not defined and likely to change in future releases. Make sure the camera position and orientation is set when the view is 
> 
I cant able to render 3d object with minimum sdk version 24

`ModelRenderable.builder().setSource(this,Uri.parse("Icon_01.sfb"));`

**Comment by malik-at-work** [comment]
Hi can you give a bit more info.  What error are you seeing and what version are you running on?

**Comment by SudarshanSudhan** [comment]
MinSDKVersion : 21
Trying to use Sceneform ModelRenderable, but it shows error like this API needs min version 24

**Comment by malik-at-work** [comment]
That is correct, version 3.1 of Sceneform requires CompletableFuture and OpenGL ES 3.1 which are not currently available in older SDK's than 24.   I'll open a feature request for supporting SDK version 21.

**Comment by GarrettStrobel** [comment]
Hello.  Any updates on this feature request?  We would like to use SceneForm as a rendering engine without the AR components but we must support min SDK 21 and OpenGL ES 2.0

**Comment by mojikashani** [comment]
We also desperately need the support for min SDK 21...

**Comment by xfort** [comment]
We also desperately need the support for min SDK 21
